pivy-importer/src/main/groovy/com/linkedin/python/importer/deps/SdistDownloader.groovy

```diff
@@ -56,6 +55,10 @@ class SdistDownloader extends DependencyDownloader {
             throw new RuntimeException("Unable to find source dist for $dep")
         }
 
+        name = IvyFileWriter.getActualModuleName(sdistDetails.filename, version)
+        dep = name + dep.substring(name.length())
```

Comment:
This seems weird, can't we just get the full dependency from fileName instead of substituting just the name here? Also, we should probably add comments that we need to do this to support case sensitive.

Reply:
We already know the version number, just need to make sure the name has correct case as PyPI, we don't need to override "dep", I made some changes and add some comments.